examples/flagging-bad-clients/hivemq/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM hivemq/hivemq4:4.16.0
+
+# postgre
+ADD extension/hivemq-postgresql-extension-0.1.0-RC-1-proguarded.zip /opt/hivemq/extensions
```

Comment:
This extension is neither public nor checked out in this project.
So either we need to mention in the quickstart, that users need to download the extension themselves from somewhere (?) and put it into the extension folder, or include the pre-release zip in this project for now. 

Will the extension be publicly downloadable? Then we can do the same as with the prometheus extension in L.14 as soon as the extension is released.

Reply:
Good point. It is not released yet. However, we will merge once released. ETA very soon :-)

Reply:
Now uses the released PostgreSQL extension.